src/components/dropdown/__spec__.js

```diff
@@ -921,9 +922,18 @@ describe('Dropdown', () => {
       });
     });
 
-    it('returns the list', () => {
-      let classes = 'carbon-dropdown__list-block carbon-dropdown__list-hidden';
-      expect(instance.additionalInputContent[1].props.className).toEqual(classes);
+    it('creates the list in a Portal', () => {
+      const wrapper = mount(
+        <Dropdown
+          name='foo'
+          options={ Immutable.fromJS([{ id: 1, name: 'foo' }, { id: 2, name: 'bar' }]) } value='1'
+        />);
+      const portal = wrapper.find(Portal);
+      expect(portal.props().children.key).toEqual('listBlock');
+
+      const portalDropdownList = wrapper.find(Portal).props().children.props.children.props;
```

Comment:
Just curious if there is a cleaner way to drill down - maybe something in enzyme to fetch the list?

Reply:
Yeah you should just be able to use `find`.

Reply:
Yip you are both right. I added a snap shot.